tests/unit/CorrectNameOptions.spec.ts

```diff
@@ -0,0 +1,59 @@
+import Vue from 'vue'
```

Comment:
Why did you add this test suite? It's not related to the safety checks you implemented...?

Reply:
Wait I added it? I don't think I added any test suites

Reply:
Should I delete it then?

Reply:
According to GitHub, yes, it's a new file in your commit:

![image](https://github.com/bcgov/business-edit-ui/assets/10002889/3cfedd35-9558-440c-ae83-895d2061db1d)

If you didn't mean to do this then please delete it in a new commit, and it will disappear from the "total" commits here.